### DISCUSSION TITLE
Error while trying to add IP with so-allow

### DISCUSSION THREAD

**Discussion by Augustin1231** [Security-Onion-Solutions/securityonion#9748] [Unsupported Versions]
Hello guys ! Having a bit of a problem here , any help would be much appreciated. 

When I am trying to add an IP address with so-allow for a Ubuntu Machine I am getting this error : 

 **_command 'salt-call state.apply -l quiet firewall queue=True' failed Security Onion_** 

Another thing is that when I try to : soup , I am also getting an error saying unable to pull from github and to check my connection but the adapter is already on NAT . Are these two connected ? 

Thank you very much in advance ! Have a great day !

**Comment by InfosecGoon** [comment]
Sounds like you've got a few problems going on. 

What version of SO are you running? Is it a standalone node? How did you install it? What's the base OS?

If you run "so-checkin", does it complete successfully?
